src/ruby/ext/grpc/extconf.rb

```diff
@@ -24,10 +24,18 @@
 
 ENV['MACOSX_DEPLOYMENT_TARGET'] = '10.7'
 
-ENV['AR'] = RbConfig::CONFIG['AR'] + ' rcs'
-ENV['CC'] = RbConfig::CONFIG['CC']
-ENV['CXX'] = RbConfig::CONFIG['CXX']
-ENV['LD'] = ENV['CC']
+if ENV['AR'].nil || ENV['AR'].size == 0
```

Comment:
Per the test failures, can you please replace `.nil` with `.nil?`

.... to fix the missing method names (`.nil?` is the correct one)

Reply:
@apolcyn could you clarify whether the remaining test failures are a problem?  I cannot identify anything in them that would be caused by this pull.